cool.py

```diff
@@ -483,9 +545,52 @@ def append_results(results_df):
     )
 
     job = BIGQUERY_CLIENT.load_table_from_dataframe(results_df, table_id, job_config=job_config, location="US")
-    job.result()  # Waits for table load to complete.
+   
+    try:
+        job_result = job.result()  # Waits for table load to complete.
+    except Exception as ex:
+        logging.error("unable to append rows into the results table! %s", ex)
+
+        return
+
+    if job_result.total_rows == 0:
```

Comment:
**question**: does this count the total rows in the table or specifically in a cursor? Would [output_rows](https://github.com/googleapis/python-bigquery/blob/075aa66a7488636606d1ac0fe21690a37904746a/google/cloud/bigquery/job/load.py#LL888C9-L888C20) be more accurate and match the input dataframe row count?

Reply:
we'll want to verify this at some point.